packages/ui-library/src/components/sidebar-navigation/mobile.js

```diff
@@ -22,7 +22,7 @@ const Mobile = ( { children, openButtonScreenReaderText = "Open", closeButtonScr
 				<Dialog.Panel className="yst-relative yst-flex yst-flex-1 yst-flex-col yst-max-w-xs yst-w-full yst-z-40 yst-bg-slate-100">
 					<div className="yst-absolute yst-top-0 yst-right-0 yst--mr-14 yst-p-1">
 						<button
-							className="yst-flex yst-h-12 yst-w-12 yst-items-center yst-justify-center yst-rounded-full focus:yst-outline-none focus:yst-bg-slate-600"
+							className="yst-flex yst-h-12 yst-w-12 yst-items-center yst-justify-center yst-rounded-full focus:yst-outline-none yst-bg-slate-600 focus:yst-ring-2 focus:yst-ring-inset focus:yst-ring-primary-500"
```

Comment:
Why the `focus:yst-ring-inset`?
The missing `focus:yst-ring-offset-2` makes sense due the background being transparent 🙈 
See the [focus style for our normal buttons](https://github.com/Yoast/wordpress-seo/blob/trunk/packages/ui-library/src/elements/button/style.css#L20:L23).

Reply:
The `focus:yst-ring-inset` is setting the border from inside the close button. 
The `focus:yst-ring-offset-2` adds solid white box-shadow and increasing the thickness.
The classes I received from @luckickken are `focus:ring-2 focus:ring-inset focus:ring-primary-500`